app/components/views/SearchView/index.tsx

```diff
@@ -70,23 +69,27 @@ const SearchView = () => {
           <ArtistsView artists={artists} inLibrary={false} showImages />
         ),
         imageUrl: `${APP_URL}/artists_icon.svg`,
-        sublabel: `${artists?.length} ${pluralize("artist", artists?.length)}`,
+        sublabel: `${artists?.length} ${
+          artists?.length === 1 ? "artist" : "artists"
+        }`,
       }),
       ...getConditionalOption(!!albums?.length, {
         type: "view",
         label: "Albums",
         viewId: viewConfigMap.albums.id,
         component: () => <AlbumsView albums={albums} inLibrary={false} />,
         imageUrl: `${APP_URL}/albums_icon.svg`,
-        sublabel: `${albums?.length} ${pluralize("album", albums?.length)}`,
+        sublabel: `${albums?.length} ${
+          albums?.length === 1 ? "album" : "albums"
```

Comment:
We should add `pluralize` to make this type of thing easier going forward

Reply:
I removed the package as part of this PR to save some bytes 😅.

I created a simplified `pluralize` helper function in https://github.com/tvillarete/ipod-classic-js/pull/152/commits/9cc49f70017dc372dd93fbe23877a63c2790d933